blog/top-5-cybersecurity-incidents-in-cicd/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Top 5 Cybersecurity Incidents in CI/CD Pipelines
+description: Explore the top 5 cybersecurity incidents in CI/CD pipelines, including the SolarWinds attack and Codecov breach. Learn how these high-profile breaches highlight critical security vulnerabilities and best practices to protect your development processes
+tags: ["Cybersecurity", "CI/CD", "Incidents", "Security"]
+date: 2024-08-19
+---
+
+### Top 5 Cybersecurity Incidents in CI/CD Pipelines
+
+CI/CD (Continuous Integration and Continuous Deployment) pipelines have revolutionized software development, enabling rapid code integration, testing, and deployment. However, their growing complexity and reliance on automated processes have also introduced significant cybersecurity risks. Below, we examine five of the highest-profile cybersecurity incidents involving CI/CD pipelines in the IT industry, highlighting the vulnerabilities and lessons learned.
+
+<!-- truncate -->
+
+---
+
+**1. SolarWinds Supply Chain Attack (2020)**  
+The SolarWinds incident is one of the most devastating software supply chain attacks in history. Attackers compromised the CI/CD pipeline, injecting malicious code into the SolarWinds Orion software, which was then distributed to over 18,000 customers, including Fortune 500 companies and U.S. government agencies.
+
+**Associated OWASP CI/CD Risks:**  
+- [CICD-SEC-1: Insufficient Flow Control Mechanisms](https://owasp.org/www-project-top-10-ci-cd-security-risks/CICD-SEC-01-Insufficient-Flow-Control-Mechanisms)  
+- [CICD-SEC-10: Insufficient Logging and Visibility](https://owasp.org/www-project-top-10-ci-cd-security-risks/CICD-SEC-10-Insufficient-Logging-And-Visibility)
+
+**References:**  
+- [SolarWinds hack timeline: Who knew what, and when (CSO Online)](https://www.csoonline.com/article/3620961/solarwinds-hack-timeline-who-knew-what-and-when.html)  
+
+---
+
+**2. Codecov Bash Uploader Incident (2021)**  
+Codecov, a code coverage tool, was compromised after attackers modified the Bash Uploader script used in its CI/CD process. The alteration allowed the exfiltration of sensitive data like credentials and environment variables from thousands of customers.
+
+**Associated OWASP CI/CD Risks:**  
+- [CICD-SEC-6: Insufficient Credential Hygiene](https://owasp.org/www-project-top-10-ci-cd-security-risks/CICD-SEC-06-Insufficient-Credential-Hygiene)  
+- [CICD-SEC-9: Improper Artifact Integrity Validation](https://owasp.org/www-project-top-10-ci-cd-security-risks/CICD-SEC-09-Improper-Artifact-Integrity-Validation)
```

Comment:
I would check the relevance of these risks after your answer to "Do we have more info about how their get access to the ci/cd pipeline ?"